### DISCUSSION TITLE
Virtual Mem Cards over FSP (GC Loader, RetroNAS)

### DISCUSSION THREAD

**Discussion by davistroy40** [emukidid/swiss-gc#677] [General]
Hello,

I just set up retronas (https://github.com/danmons/retronas) in a debian11 VM under proxmox. FSP for games seems to be working great. I can access all my ISOs over my network! Still have to play around with SMB and FTP but that can be another project for another day.

My question is, are virtual memory cards supported while running games over FSP? Correct me if I am wrong but, from what I have read, mem cards and the broadband adapter run over the same bus so it would make sense that there is a limitation. I can confirm that emulated cards work fine for games loaded from GCLoader SD card.

For context, I am running GCLoader (most recent BETA firmware) and Swiss v0.6 rev:1318. BBA FSP over RetroNAS FSP (http://fsp.sourceforge.net/) in debian11 VM. I am aware that it is currently advised to use emulated memcards with a SP2SD2 rather than GC loader. But since I have not been playing very much, I figured it is okay for the time being. Please advise if you want me to test with emulated memcards under SP2SD2.

Thanks!

**Comment by davistroy40** [comment]
Also, for additional context my ProxMox box is a AMD 2200g with 8gb RAM. The debian VM has 3GB RAM, 4 processors and 150GB allocated. I am not running a raspberry pi :)

**Comment by Extrems** [comment]
They are not. The FSP protocol isn't designed to modify files, and there isn't enough memory to use SD cards instead.

**Reply by davistroy40** [reply]
Ahh that would make sense. Thanks for the info!
